iTerm/zsh.md

```diff
@@ -94,7 +94,7 @@ git clone --recursive https://github.com/sorin-ionescu/prezto.git "${ZDOTDIR:-$H
 
 Next create your `~/.zshrc` by running:
 
-```sh
+```console
```

Comment:
```suggestion
```sh
```

Reply:
I would like to keep this as console since this is a post install command. Let me know if you have a differing opinion.. will change it. For now i would like to maintain this idea.

Reply:
As before, I think `sh` looks better:

# `sh`
```sh
setopt EXTENDED_GLOB
for rcfile in "${ZDOTDIR:-$HOME}"/.zprezto/runcoms/^README.md(.N); do
    ln -s "$rcfile" "${ZDOTDIR:-$HOME}/.${rcfile:t}"
  done
```
# `console`
```console
setopt EXTENDED_GLOB
for rcfile in "${ZDOTDIR:-$HOME}"/.zprezto/runcoms/^README.md(.N); do
    ln -s "$rcfile" "${ZDOTDIR:-$HOME}/.${rcfile:t}"
  done
```